myapp/app/models/user.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  require 'bcrypt'
+  attr_accessor :password, :password_confirmation
+
+  has_many :tasks, dependent: :nullify
```

Comment:
Why is dependent nullify?

Reply:
I would like to keep their tasks when deleting users, change the user of those tasks to nil, so it's easier to manage all the tasks

Reply:
I see. Can you implement the test?

Reply:
I found out yesterday that this is from step18, so I will remove dependent and it and the test will be submitted in the pr of step18.